assets/constants.ts

```diff
@@ -145,13 +145,19 @@ export default {
     "I18N_TOPIC_dLmjjMDbCcrf_TITLE",
     "I18N_TOPIC_0abdeaJhmfPm_TITLE",
     "I18N_TOPIC_5g0nxGUmx5J5_TITLE",
+    "I18N_TOPIC_VjCpR1DB2MVN_TITLE",
+    "I18N_TOPIC_F4vMtwKMS6jR_TITLE",
+    "I18N_TOPIC_5cbCOcglPdrn_TITLE",
     "I18N_TOPIC_iX9kYCjnouWN_DESCRIPTION",
     "I18N_TOPIC_sWBXKH4PZcK6_DESCRIPTION",
     "I18N_TOPIC_C4fqwrvqWpRm_DESCRIPTION",
     "I18N_TOPIC_qW12maD4hiA8_DESCRIPTION",
     "I18N_TOPIC_dLmjjMDbCcrf_DESCRIPTION",
     "I18N_TOPIC_0abdeaJhmfPm_DESCRIPTION",
     "I18N_TOPIC_5g0nxGUmx5J5_DESCRIPTION",
+    "I18N_TOPIC_VjCpR1DB2MVN_DESCRIPTION",
+    "I18N_TOPIC_F4vMtwKMS6jR_DESCRIPTION",
+    "I18N_TOPIC_5cbCOcglPdrn_DESCRIPTION",
     "I18N_EXPLORATION_K645IfRNzpKy_TITLE",
     "I18N_EXPLORATION_K645IfRNzpKy_DESCRIPTION",
     "I18N_EXPLORATION_Knvx24p24qPO_TITLE",
```

Comment:
Just wanted to ask why do we have a bit of inconsistency in the pattern? I mean in the above lines we have separated the "TITLE" and "DESCRIPTION" for the `topics` and in the below `exploration` section we have them in the alternate pattern.

Reply:
I have no idea, haha. I just followed what was there already :)

Reply:
Yep! Sounds good.
Thanks